jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/snmp/SnmpNTCIP1218Protocol.java

```diff
@@ -86,9 +86,10 @@ public static VariableBinding getVbRsuMsgRepeatPriority(int index) {
     }
 
     public static VariableBinding getVbRsuMsgRepeatOptions(int index) {
+        byte[] val = {(byte) 0xc0};
```

Comment:
This is just defining the byte string in a more correct way, right?

Reply:
Yep, that just ensures that the value of the byte string is C0